corehq/apps/hqwebapp/static/hqwebapp/js/django.js

```diff
@@ -3,5 +3,5 @@
     Allow usage of gettext, RequireJS-style, rather than accessing the global django.
 */
 define([], function () {
-    return { gettext: django.gettext };
+    return { gettext: window.django ? django.gettext : undefined };
```

Comment:
This looks like a contingency for tests in production code. I wonder if there's a way to make a shim for `window.django` when running tests?

Reply:
Made an attempt in 308eb861e23, let's see if tests still pass 🤞

Reply:
tests pass!